### PR TITLE
Add recipe for wgetter 0.7

### PR DESCRIPTION
**Pull request by npavlovikj**
Add recipe for wgetter 0.7

### CODE REVIEW

**Comment by npavlovikj** [comment]
@bgruening , "wgetter" is a dependency for a bioconda recipe I am working on, so can you please review this one when you have a chance? Thanks!

**Comment by synapticarbors** [comment]
I'm happy to merge this once I have confirmation that all of the people listed as co-maintainers are ok with being listed

**Comment by npavlovikj** [comment]
Thanks for replying @synapticarbors ! I modified the maintainers list with only my name just to be safe, so please go ahead and merge the PR when you have a chance.

File: recipes/wgetter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+test:
+  imports:
+    - wgetter
+ 
+about:
+  home: https://github.com/phoemur/wgetter
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (jakirkham) [inline recipes/wgetter/meta.yaml]: Added PR ( https://github.com/phoemur/wgetter/pull/4 ) upstream to get the license file packaged in the PyPI `sdist`.

**Pull request merged**